release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Initial release with python 3
```

Comment:
```suggestion
* Initial release as a community-supported app.
```

Reply:
Generally, we are not adding this
<img width="454" alt="Screenshot 2021-12-10 at 2 45 14 PM" src="https://user-images.githubusercontent.com/64279254/145548848-ad03af69-7853-43b6-a55a-f347cfcb704c.png">